src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -83,25 +87,23 @@ def update_event_response(
     """
     if is_lumigo_edge(host):
         return
-    last_event = SpansContainer.get_span().pop_last_span()
+    last_event = SpansContainer.get_span().pop_span(span_id)
     if last_event:
         http_info = last_event.get("info", {}).get("httpInfo", {})
         if not host:
             host = http_info.get("host", "unknown")
-        else:
-            HttpState.previous_response_body = b""
+        body = http_info.get("response", {}).get("body", "").encode() + body
```

Comment:
Looks like now we always concat the `body` but there should be ` limit, looks like you deleted this part
![image](https://user-images.githubusercontent.com/49228804/128876138-5cdfa42a-35c6-4572-9e29-cc6d543524f9.png)

Reply:
nope, this part exists inside `parser.parse_response`. It was here because we used to save the body in two places (both the span, and inside the HttpState), and now we save it only in the span :)